src/pages/index.js

```diff
@@ -24,7 +23,6 @@ export default function HomePage() {
 							id={teaser._id}
```

Comment:
Attention: This comment is about the mapping method 4 lines above.

You should structuring your variables in away that any structure like data.data could not result. Any other developer will not be able to understand what is behind this.